packages/blocks/src/store/reducer.js

```diff
@@ -66,16 +65,6 @@ export function blockTypes( state = {}, action ) {
  */
 export function blockStyles( state = {}, action ) {
 	switch ( action.type ) {
-		case 'ADD_BLOCK_TYPES':
-			return {
-				...state,
-				...mapValues( keyBy( action.blockTypes, 'name' ), ( blockType ) => {
-					return uniqBy( [
-						...get( blockType, [ 'styles' ], [] ),
-						...get( state, [ blockType.name ], [] ),
-					], ( style ) => style.name );
-				} ),
-			};
```

Comment:
I know people used to disable styles by removing them from this reducer, this won't be possible anymore with this change.

Reply:
Interesting. Thanks for sharing.

Reply:
> I know people used to disable styles by removing them from this reducer, this won't be possible anymore with this change.

Can you elaborate on what you mean with this? Any links you can share with more information?